app/assets/javascripts/hyrax/file_manager.es6

```diff
@@ -5,7 +5,7 @@ export default class FileManager {
   constructor() {
     this.save_manager = this.initialize_save_manager()
     this.sorting()
-    this.save_affix()
+//    this.save_affix()
```

Comment:
@wang58 Why is this line commented out, and retained?

Reply:
Removed the line.